parser.py

```diff
@@ -190,8 +104,14 @@ class QASMToStringTransformer(Transformer):
 
     def mainprogram(self, *args):
         args = unpack(args)
-        version, program_str = args
-        header = "OPENQASM {};".format(version)
+
+        if len(args) == 1:
+            program_str = args[0]
+            header = ""
```

Comment:
This is required because the `qelib1.inc` file, when included, does not contain the `OPENQASM 2.0` specifier.

We _could_ do this recursively (somehow attach the parsed tree from the included file to the existing tree?) but I'm not sure how to do that.